components/Table/Table.js

```diff
@@ -95,12 +113,184 @@ export class Table {
     this.tableHeader.append(tableHeaderRow);
   }
 
+  renderDropDownMenu(menuBtn, isSortable, cell) {
+    const dropDownMenu = document.createElement('div');
+    dropDownMenu.classList.add('drop-down__menu');
+    this.menuDropDownItems = this.menuConfig
+      .filter((elConfig) => {
+        if (
+          this.callbacksMap.sortCallback ||
+          this.callbacksMap.filterCallback
+        ) {
```

Comment:
Write in 1 line please

Reply:
How `filterCallback` related to the sorting?

Reply:
If you're going to check, that we have a callback for filtering, I guess we should a bit separate our conditions